test/cross_section_test.cpp

```diff
@@ -167,3 +169,24 @@ TEST(CrossSection, FillRule) {
   CrossSection nonZero(polygon, CrossSection::FillRule::NonZero);
   EXPECT_NEAR(nonZero.Area(), 0.875, 0.001);
 }
+
+TEST(CrossSection, Hull) {
+  auto circ = CrossSection::Circle(10);
+  auto circs = std::vector<CrossSection>{circ, circ.Translate({0, 30}),
+                                         circ.Translate({30, 0})};
+  auto circ_tri = CrossSection::Hull(circs);
+  auto centres = SimplePolygon{{0, 0}, {0, 30}, {30, 0}, {15, 5}};
+  auto tri = CrossSection::Hull(centres);
+  auto circ_tri_ex = Manifold::Extrude(circ_tri, 10);
+
+#ifdef MANIFOLD_EXPORT
+  if (options.exportModels)
+    ExportMesh("cross_section_hull_circ_tri.glb", circ_tri_ex.GetMesh(), {});
+#endif
+
+  auto circ_area = circ.Area();
+  EXPECT_FLOAT_EQ(circ_area, (circ - circ.Scale({0.8, 0.8})).Hull().Area());
+  EXPECT_FLOAT_EQ(
+      circ_area * 2.5,
```

Comment:
Where does the 2.5 come from?

Reply:
The hull triangle cuts a quarter out of the circle on the origin, and an eight out of the other two.

Reply:
ah, as in 3 - 0.5 = 2.5 - I get it now, thanks.